### PR TITLE
DEVDOCS-3922: [net new] Create POS Foundation overview

### PR DESCRIPTION
**Pull request by bc-traciporter**
# [DEVDOCS-3922](https://bigcommercecloud.atlassian.net/browse/DEVDOCS-3922)

## What changed?
* thing_that_changed

### CODE REVIEW

**Comment by slsriehl** [comment]
@traci707 are you waiting for notes from me on this?

**Review by stanzikratelbc — COMMENTED** [review]
Made some minor changes to this document to improve readability.

File: docs/api-docs/partner/pos/pos-foundation-overview.md [hunk 1]
```diff
@@ -0,0 +1,253 @@
+# POS Foundation
+
+POS Foundation provides a proof-of-concept [open-source framework](https://github.com/bigcommerce/point-of-sale-foundation) that helps developers deliver custom point-of-sale (POS) checkout interfaces to BigCommerce merchants with brick-and-mortar locations. It provides a template to scaffold custom POS solutions that use secure, pre-certified EMV card readers. POS Foundation's default integration with Stripe Terminal helps significantly accelerate development time. 
+
+POS Foundation is a manual connector app that uses a store API account. The app will not be displayed in the store control panel. POS Foundation does not require that you create an app profile or use ngrok. This proof-of-concept tutorial only applies to local network operations and in its current state does not create its own channel. We are looking to expand this foundation and would like to collaborate with interested agencies.
```
Comment (slsriehl) [inline docs/api-docs/partner/pos/pos-foundation-overview.md:5]:
```suggestion
POS Foundation scaffolds manual connector apps that use [store API accounts](/api-docs/getting-started/authentication/rest-api-authentication#store-api-accounts). You do not need to create an app profile or use ngrok. The app will not appear in the store control panel and is only accessible to computers running on the same local network as the app's server. In contrast to [Subscription Foundation](/api-docs/partner/subscription-solutions/foundation-guide)-derived apps, POS Foundation apps do not create their own dedicated sales channels.  If you're interested in contributing to this package, >>> what did we want them to do? submit a PR? contact partner rep?

```

File: docs/api-docs/partner/pos/pos-foundation-overview.md [hunk 2]
```diff
@@ -0,0 +1,253 @@
+# POS Foundation
+
+POS Foundation provides a proof-of-concept [open-source framework](https://github.com/bigcommerce/point-of-sale-foundation) that helps developers deliver custom point-of-sale (POS) checkout interfaces to BigCommerce merchants with brick-and-mortar locations. It provides a template to scaffold custom POS solutions that use secure, pre-certified EMV card readers. POS Foundation's default integration with Stripe Terminal helps significantly accelerate development time.
```
Comment (slsriehl) [inline docs/api-docs/partner/pos/pos-foundation-overview.md:3]:
```suggestion
POS Foundation provides a proof-of-concept [open source framework](https://github.com/bigcommerce/point-of-sale-foundation) for building point-of-sale (POS) checkout interfaces that serve BigCommerce merchants with physical locations. It provides a template to scaffold custom POS solutions that use secure, pre-certified EMV card readers. POS Foundation's default integration with Stripe Terminal can significantly accelerate development time. 
```

File: docs/api-docs/partner/pos/pos-foundation-overview.md [hunk 3]
```diff
@@ -0,0 +1,253 @@
...
+
+## Configure accounts
+
+<!-- theme: info -->
+> #### Account configuration requirements
+> We recommend that you use a sandbox store that has the same multi-storefront status as the production store. For example, if you're developing for a multi-storefront-enabled merchant store, use a multi-storefront sandbox. For information on configuring multi-storefront, see [Multi-Storefront](https://support.bigcommerce.com/s/article/Multi-Storefront?language=en_US).
+
+To configure your accounts, complete the following steps:
+
+1. [Create a BigCommerce Sandbox Store](/api-docs/partner/getting-started/create-a-sandbox-store?source=subscription-foundation) to develop and test your apps.
```
Comment (slsriehl) [inline docs/api-docs/partner/pos/pos-foundation-overview.md:25]:
```suggestion
1. To develop and test safely, you need a BigCommerce sandbox store. If you don't have one, [Create a Sandbox Store](/api-docs/partner/getting-started/create-a-sandbox-store?source=pos-foundation).
```

Comment (slsriehl) [reply docs/api-docs/partner/pos/pos-foundation-overview.md:25]: parallel with Subscription Foundation

**Comment by slsriehl** [comment]
I started making comments, but I'm going to make a dedicated suggestion branch.

File: docs/api-docs/partner/pos/pos-foundation-overview.md [hunk 4]
```diff
@@ -0,0 +1,253 @@
...
+6. Sign in to the app as an admin at `http://localhost:3000/signin`. The default admin PIN in the seed data is `1234`.
+
+![pos-sign-in](https://storage.googleapis.com/bigcommerce-production-dev-center/images/pos-sign-in.png)
+
+7. After you access Prisma Studio, navigate to the **Settings** screen to save your store address, which the app uses for checkout tax calculations. If you fail to supply the store address, you will receive the following error:
+  
+![pos-missing-store-address](https://storage.googleapis.com/bigcommerce-production-dev-center/images/POS-missing-store-address.jpeg)
+
+
+## Connecting to a reader in the POS
```
[the remark this answers is not in the document]
Comment (slsriehl) [reply docs/api-docs/partner/pos/pos-foundation-overview.md]: linked to stripe reference. done.

**Comment by slsriehl** [comment]
I merged my suggestion branch into this feature branch. I also neutralized some stray changes that belong on another branch.

**Review by slsriehl — APPROVED** [review]
🎉🎉🎉

**Pull request merged**